spec/factories/user.rb

```diff
@@ -19,5 +18,15 @@
     confirmed_at { Time.zone.now }
     created_at { Time.zone.now }
     updated_at { Time.zone.now }
+
+    transient do
+      old_user_id { SecureRandom.uuid }
+    end
+
+    after(:create) do |user, evaulator|
```

Comment:
Should this be `evaluator`?

Reply:
Good spot!!

Reply:
Fixed in [0547e55](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/894/commits/0547e55f57cdb6f8426f07bf814755dc0957e86d)